main.rb

```diff
@@ -1,11 +1,14 @@
-require_relative './lib/person'
-require_relative './decorators/decorator'
+#!/usr/bin/env ruby
+require_relative 'app'
 
-person = Person.new(22, 'maximilianus')
-puts person.correct_name
+def main
+  app = App.new
+  app.run
```

Comment:
[REQUIRED] The content of the `run` method should be present in this file instead of just calling it here. These are things like presenting users with a list of options, allowing users to select an option, have a way to quit.

Reply:
@fobadara Thanks but I think it was not part of the requirement.